src/components/LandingPage/ContentLayout.tsx

```diff
@@ -0,0 +1,66 @@
+'use client';
+
+import { ReactNode, useEffect, useRef, useState } from 'react';
+
+interface Props {
+  title: ReactNode;
+  btnBoldText: string;
+  children: ReactNode;
+}
+
+/**
+ * 스크롤 애니메이션이 들어가는 content의 layout
+ * @param children title과 button 제외 body 영역
+ */
+function ContentLayout({ title, btnBoldText, children }: Props) {
+  const targetRef = useRef(null);
+  const sectionRef = useRef(null);
+  const [isVisible, setIsVisible] = useState(true);
+
+  useEffect(() => {
+    const observerOptions = {
+      root: null,
+      rootMargin: '0px',
+      threshold: 0,
+    };
+
+    const observer = new IntersectionObserver(entries => {
+      entries.forEach(entry => {
+        setIsVisible(entry.isIntersecting);
+      });
+    }, observerOptions);
+
+    if (targetRef.current) {
+      observer.observe(targetRef.current);
+    }
+
+    return () => {
+      if (targetRef.current) {
+        observer.unobserve(targetRef.current);
+      }
+    };
+  }, []);
+
+  return (
+    <div className='ml-[20%] max-w-[712px]'>
+      <div ref={targetRef}></div>
+      <section
+        ref={sectionRef}
+        className={['flex w-full flex-col gap-10', isVisible ? '' : 'animate-content-out opacity-0'].join(' ')}
```

Comment:
개인적인 의견으로는 tailwind만 사용하실 경우에 isVisible인 경우에만 className이 필요하고 아닌 경우에 필요없다면
```suggestion
        className = {`flex w-full flex-col gap-10 ${isVisible && 'animate-content-out opacity-0}`}
```
요렇게 작성해도 괜찮을 것 같습니다